appuio/haproxy/templates/service.yaml

```diff
@@ -14,16 +14,10 @@ spec:
       targetPort: {{ .Values.haproxy.frontendPort }}
       protocol: TCP
       name: frontend
-    {{- if or .Values.haproxy.redisk8s.metrics.enabled .Values.haproxy.galerak8s.metrics.enabled}}
```

Comment:
Could you make this configurable? It would make sense to allow exposing this through the service conditionally.

Reply:
Added a new parameter to make this configurable